Dynamic-Programming/tests/KadaneAlgo.test.js

```diff
@@ -0,0 +1,8 @@
+const fc = require('../kadaneAlgo')
```

Comment:
```suggestion
 import { kadaneAlgo } '../KadaneAlgo'
```
Use ESM and the correct file name.